web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -287,6 +287,7 @@ export namespace MenuOverlay {
           onFirstUpdate: async () => {
             // We need to find all focusable elements, after Popper finish its positioning calculation
             if (this.isOpen) {
+              this.popperInstance?.update();
```

Comment:
This fixes the issue mentioned in the PR description.
Known issue: https://stackoverflow.com/questions/46476857/popper-js-wrong-initial-position-of-dropdown-tooltip